spec/controllers/v0/vic/profile_photo_attachments_controller_spec.rb

```diff
@@ -72,10 +35,45 @@
           }
         )
 
-        expect(data).to have_key('filename')
-        expect(data['path']).to eq "profile_photo_attachments/#{InProgressForm.last.id}"
         expect(InProgressForm.count).not_to eq(before_count)
       end
     end
   end
+
+  describe '#show' do
```

Comment:
Would like to see a spec for invalid guid, including attempt at path/directory traversal via the guid (which I realize doesn't work in S3 but it's something bad dudes might try)

Reply:
The GUID isn't actually used for any path manipulation/lookup/traversal. That comes entirely from the `file_data` column on the db, which is looked up by the GUID. However, I will definitely add some invalid guid tests to verify the behavior.

Reply:
Spec added here: https://github.com/department-of-veterans-affairs/vets-api/pull/1703/files#diff-cbbd304f6fe51adecbd198b8c57a0886R79